var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -122,7 +122,7 @@ class Flecsi(CMakePackage, CudaPackage):
     depends_on("legion+hdf5", when="backend=legion +hdf5 @2.0:")
     depends_on("legion +kokkos +cuda", when="backend=legion +kokkos +cuda @2.0:")
     depends_on("hdf5@1.10.7:", when="backend=legion +hdf5 @2.0:")
-    depends_on("hpx@1.3.0 cxxstd=17 malloc=system", when="backend=hpx @2.0:")
+    depends_on("hpx@1.8.1 cxxstd=17 malloc=system", when="backend=hpx @2.0:")
```

Comment:
This should allow 1.8.1 and newer:
```suggestion
    depends_on("hpx@1.8.1: cxxstd=17 malloc=system", when="backend=hpx @2.0:")
```

Reply:
@rbberger thanks for this! I also think that we should make it depend on the soon to be released HPX V1.9.1 instead.

Reply:
Ok, but in any case please add the `:` to future proof it.

Reply:
@hkaiser @rbberger I tried to build 2.x with +hpx but despite having checked that hpx-1.8.1 / 1.9.1 are in the spack build of flecsi, the CMake variable FOUND_MPX was not set:
```py
  >> 61    CMake Error at cmake/hpx.cmake:30 (message):
     62      HPX is required for this configuration
```
Also it seems that flecsi-2.2.1 does not have the include files it likes to include in it's tree:
```c++
  >> spack-src/flecsi/run/backend.hh:24:10: fatal error: hpx/context.hh: No such file or directory
     100       24 | #include "hpx/context.hh"
     101          |          ^~~~~~~~~~~~~~~~
spack-src/flecsi/exec/backend.hh:58:10: fatal error: flecsi/exec/hpx/policy.hh: No such file or directory
     108       58 | #include "flecsi/exec/hpx/policy.hh"
     109          |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I have updated the package to HPX V1.91. now.